cypress/e2e/2-advanced-examples/storage.cy.js

```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
+
+context('Local / Session Storage', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:8080/commands/storage')
+  })
+  // Although local storage is automatically cleared
```

Comment:
Thinking throughout we should use `localStorage` and `sessionStorage` to match how these are stored on window

Reply:
Addressed in https://github.com/cypress-io/cypress-example-kitchensink/pull/579/commits/7e58c1d53861c2719a6cb2ae191e526ab990723c